cookiecutter_context.json

```diff
@@ -0,0 +1,60 @@
+{
+    "formal_name": [
+        "First, we need a formal name for your application. ",
+        "This is the name that will be displayed to humans whenever the ",
+        "name of the application is displayed. It can have spaces and ",
+        "punctuation if you like, and any capitalization will be ",
+        "used as you type it. "
+    ],
+    "app_name": [
+        "Next, we need a name that can serve as a machine-readable ",
+        "Python package name for your application. ",
+        "\n",
```

Comment:
Does this need the `\n`? Won't this render as a double newline?

Reply:
No, there's no new line unless you explictly specify one with `\n`.
I've used `"".join(explanation[key])` to join these into a single string.